### DISCUSSION TITLE
Анонимный доступ

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#99] [Есть идея, а что если...]
Можно ли открыть доступ по ссылке на какую либо таблицу? 
Если да, то на какие и каким образом?

**Comment by ttm-alexeypolunin** [comment]
В MIT это возможно только к [анонимным таблицам](https://ru.docs.totum.online/anonymous-tables). Доступ по авторизационным ссылкам будет в PRO-версии.
